src/isDisplayable.js

```diff
@@ -0,0 +1,11 @@
+const isDisplayable = (button) => {
+  const numOrDecimal = button.type === "key";
```

Comment:
Ah okay, I tried to pass in button.key into it and changed button.type to what is below instead, but it has trouble parsing it it seems because it still throws the same error of `Binding member expression`.

`  const numOrDecimal = button.key === "." || button.key === "0" || button.key === "1" || button.key === "2" || button.key === "3" || button.key === "4" || button.key === "5" || button.key === "6" || button.key === "7" || button.key === "8" || button.key === "9";`

Reply:
@grimxreaper Right. In order to change what you pass to the function, you have to update both the function internals, and the calling site at the same time. You can't pass in `button.key` without updating the function internals to expect only `key` coming in. I would suggest not changing the function argument until you no longer depend on `button.type` inside the function.

Reply:
That's why I updated the function internals to button.key, and then expected it to work once I passed in button.key. /: I must be missing something about your explanation. I expected it to work since it no longer relies on button.type, if I am only using button.key inside the function (and thus passing in button.key).

Reply:
@grimxreaper You can't pass in `button.key` until the inside of the function doesn't say `button.key` but instead says something like `key`. This is really two steps. 1) Remove any usage of button inside the function which isn't `button.key`. 2) Pass in `button.key` instead of `button` which requires updating the function internals. Don't try to combine these steps.

Reply:
@grimxreaper Does it makes sense that you lose your reference to `button` when you pass in `button.key` and therefor get an error?

Reply:
@grimxreaper Lastly, any way you can simplify that code for `numOrDecimal`? Maybe using `includes` or regex?

Reply:
Can you spot there error in this code? Is it easier to see?
```is
const button = { key: "." }

isDisplayable(button.key)

function isDisplayable(button) {
   console.log(button.key)
   // .... assume the rest of the code is here.
}
```

Reply:
Have you tried using `console.log` or `debugger`? Hoping you try to debug some of these errors.

Reply:
@grimxreaper I think I see another issue with the `numOrDecimal` pseudo code after looking at `keypad.js`

Reply:
I think the change I made with the latest commit is a lot cleaner than what I had and reads well to me, let me know if that isn't satisfactory, or not what you had in mind (I think it's different). Also, would a better commit message be "simplify isDisplayable, pressedAnotherOperator arguments" ? Thank you! 🙌🏼

Reply:
@grimxreaper that is a clean commit. It's exactly what I was looking for. The commit message you suggested in the last comment is great! Why didn't that end up being the message? Can we got rebase to fix it?